profiles/default/bsd/package.use

```diff
@@ -0,0 +1,8 @@
+# Copyright 1999-2017 Gentoo Foundation.
+# Distributed under the terms of the GNU General Public License, v2
+# $Id$
+
+# Michał Górny <mgorny@gentoo.org> (26 Jan 2017)
+# Preserve the old defaults on *BSD systems.
+sys-libs/libcxx -libcxxabi libcxxrt -libunwind
+sys-libs/libcxxrt -libunwind
```

Comment:
I think preserving old defaults is not needed.
It is extremely important on clang profiles but on "standard" profiles the compiler is the same gcc as on linux, so I think it's better to keep the same clang as on linux too